mock/el8.cfg

```diff
@@ -75,15 +75,15 @@ gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Officia
 
 [foreman]
 name=foreman
-baseurl=https://yum.theforeman.org/nightly/el8/x86_64/
+baseurl=https://yum.theforeman.org/3.15/el8/x86_64/
```

Comment:
Why are we changing this on rpm/develop? Open the PR against 3.15

Reply:
retargeted